src/main/java/Duke.java

```diff
@@ -6,5 +8,72 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        // Greeting Message
+        System.out.println("Hello, I'm Duke!");
+        System.out.println("What can I do for you?");
+
+        Scanner sc = new Scanner(System.in);
+        int inputs = 0;
+        Task[] list = new Task[100];
+        while(true) {
+            String input = sc.next();
+            // Exit Command
+            if(input.equals("bye")) {
```

Comment:
This if else block is a bit long. You could extract methods for the individual commands to improve readability :D